ui/src/utils/timeSeriesTransformers.ts

```diff
@@ -25,64 +21,22 @@ interface TimeSeriesToTableGraphReturnType {
   sortedLabels: Label[]
 }
 
-export const timeSeriesToDygraph = (
+export const timeSeriesToDygraph = async (
   raw: TimeSeriesServerResponse[],
   pathname: string = ''
-): TimeSeriesToDyGraphReturnType => {
-  const isTable = false
-  const isInDataExplorer = pathname.includes('data-explorer')
-  const {sortedLabels, sortedTimeSeries} = groupByTimeSeriesTransform(
-    raw,
-    isTable
+): Promise<TimeSeriesToDyGraphReturnType> => {
+  const result = await manager.timeSeriesToDygraph(raw, pathname)
+  const {timeSeries} = result
+  const newTimeSeries = fastMap<DygraphValue[], DygraphValue[]>(
+    timeSeries,
+    ([time, ...values]) => [new Date(time), ...values]
```

Comment:
it would be nice to do this inside the worker. does Dygraph accept Unix times as dates? or strings? I think it accepts strings at least

Reply:
@121watts says it probably accepts numbers (ms since epoch).

Reply:
tried both strings and unix timestamps and neither worked 😞